scripts/pdf-template.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Probably add a ```lang="EN"```  attrib but then this is not (yet) distributed HTML and am not sure about effet on PDF so I won't be picky :)

Reply:
Running through the PDF meta doesn't show any difference, but still good practice. Updated.